### DISCUSSION TITLE
Inconsistent Results When Training to Likeness

### DISCUSSION THREAD

**Discussion by MysticSpiderman** [TheLastBen/fast-stable-diffusion#696] [General]
Hmmm. Wondering if anyone can help me solve this. 

So I am working on training a model to someones likeness. I did it yesterday and it worked pretty good. Was running prompts and it was looking how I wanted it to. 

Today I ran another session using the same base images and steps but this time every time I generate the output images look more or less like the original photos of the person. 

Any idea how to fix this or why this would happen? 

Is it possible that sometimes you can train a model and it can be a dud? Should I try retraining with the same steps? 

Curious if anyone has any insight to help resolve this issue. Thanks in advance.

**Comment by GuileGaze** [comment]
I'm also having the same problem (#688). From the little bit of research that I've done since my post, I _think_ the issue is caused if you don't have regularization images to help train the model. Take this with a grain of salt though, I literally just starting learning about Dreambooth and training custom models yesterday.

**Comment by TheLastBen** [comment]
you can resume the training for additional steps if the results are not right

**Comment by MysticSpiderman** [comment]
Thanks. 
For the most part is this caused by not enough training?
Strangly enough - i was able to still get results by NOT including my file name in my prompt. 
It was drawing on my model's likeness. And then doing that plus img2img I was able to beginning getting it closer to the likeness desired, and then did some inpainting and eventually got there. 
Though I wouldn't want to have to do this all the time.
